src/main/java/it/gov/pagopa/pu/pagopapayments/util/PagoPaNodeFaults.java

```diff
@@ -0,0 +1,29 @@
+package it.gov.pagopa.pu.pagopapayments.util;
```

Comment:
Could it be better to introduce and enums package instead?
```suggestion
package it.gov.pagopa.pu.pagopapayments.enums;
```

Reply:
done